<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParonamaFrameWidget</class>
 <widget class="QWidget" name="ParonamaFrameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParonamaFrameWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="imageAreaWidget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(47, 79, 79);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,1" columnstretch="1,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="CParonamaSourceImageDisplayerWidget" name="sourceImageWidget">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="CParonamaSagittalImageDisplayerWidget" name="sagittalImageWidget">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="CParonamaCPRImageDisplayerWidget" name="cprImageWidget">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="CParonamaVRImageDisplayerWidget" name="vrImageWidget">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="toolsMenu" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(47, 79, 79);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>25</number>
      </property>
      <item row="2" column="0">
       <widget class="CIconButton" name="windowLevelButton">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;窗宽窗位&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:enabled {border-image: url(:/StarLightStation/icon/wl.png);}
QPushButton:enabled:pressed {border-image: url(:/StarLightStation/icon/wl-.png)}
QPushButton:hover {border-image: url(:/StarLightStation/icon/wl-m.png)}
QPushButton:disabled {border-image:url(:/StarLightStation/icon/wl-no.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="CIconButton" name="rectAnnotationButton">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;矩形测量&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:enabled {border-image: url(:/StarLightStation/icon/juxing.png);}
QPushButton:enabled:pressed {border-image: url(:/StarLightStation/icon/juxing-.png)}
QPushButton:hover {border-image: url(:/StarLightStation/icon/juxing-m.png)}
QPushButton:disabled {border-image:url(:/StarLightStation/icon/juxing-no.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="CIconButton" name="moveImageButton">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;平移&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:enabled {border-image: url(:/StarLightStation/icon/pos.png);}
QPushButton:enabled:pressed {border-image: url(:/StarLightStation/icon/pos-.png)}
QPushButton:hover {border-image: url(:/StarLightStation/icon/pos-m.png)}
QPushButton:disabled {border-image:url(:/StarLightStation/icon/pos-no.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="CIconButton" name="resetImageButton">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;重置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:enabled {border-image: url(:/StarLightStation/icon/reset.png);}
QPushButton:enabled:pressed {border-image: url(:/StarLightStation/icon/reset-.png)}
QPushButton:hover {border-image: url(:/StarLightStation/icon/reset-m.png)}
QPushButton:disabled {border-image:url(:/StarLightStation/icon/reset-no.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="CIconButton" name="curveAnnotationButton">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;曲线测量&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:enabled {border-image: url(:/StarLightStation/icon/connect.png);}
QPushButton:enabled:pressed {border-image: url(:/StarLightStation/icon/connect-.png)}
QPushButton:hover {border-image: url(:/StarLightStation/icon/connect-m.png)}
QPushButton:disabled {border-image:url(:/StarLightStation/icon/connect-no.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="CIconButton" name="deleteAnnotationButton">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;删除标注&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:enabled {border-image: url(:/StarLightStation/icon/deleteAnn.png);}
QPushButton:enabled:pressed {border-image: url(:/StarLightStation/icon/deleteAnn-.png)}
QPushButton:hover {border-image: url(:/StarLightStation/icon/deleteAnn-m.png)}
QPushButton:disabled {border-image:url(:/StarLightStation/icon/deleteAnn-no.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="CIconButton" name="angleAnnotationButton">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;角度测量&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:enabled {border-image: url(:/StarLightStation/icon/angle.png);}
QPushButton:enabled:pressed {border-image: url(:/StarLightStation/icon/angle-.png)}
QPushButton:hover {border-image: url(:/StarLightStation/icon/angle-m.png)}
QPushButton:disabled {border-image:url(:/StarLightStation/icon/angle-no.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="CIconButton" name="selectImageButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;选择影像&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:enabled {border-image: url(:/StarLightStation/icon/def.png);}
QPushButton:enabled:pressed {border-image: url(:/StarLightStation/icon/def-.png)}
QPushButton:hover {border-image: url(:/StarLightStation/icon/def-m.png)}
QPushButton:disabled {border-image:url(:/StarLightStation/icon/def.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="CIconButton" name="resiezeImageButton">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;缩放&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:enabled {border-image: url(:/StarLightStation/icon/size.png);}
QPushButton:enabled:pressed {border-image: url(:/StarLightStation/icon/size-.png)}
QPushButton:hover {border-image: url(:/StarLightStation/icon/size-m.png)}
QPushButton:disabled {border-image:url(:/StarLightStation/icon/size-no.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="CIconButton" name="pologonAnnotationButton">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;多边形测量&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:enabled {border-image: url(:/StarLightStation/icon/cutline.png);}
QPushButton:enabled:pressed {border-image: url(:/StarLightStation/icon/cutline-.png)}
QPushButton:hover {border-image: url(:/StarLightStation/icon/cutline-m.png)}
QPushButton:disabled {border-image:url(:/StarLightStation/icon/cutline-no.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="CIconButton" name="lineAnnotationButton">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;直线测量&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:enabled {border-image: url(:/StarLightStation/icon/ruler.png);}
QPushButton:enabled:pressed {border-image: url(:/StarLightStation/icon/ruler-.png)}
QPushButton:hover {border-image: url(:/StarLightStation/icon/ruler-m.png)}
QPushButton:disabled {border-image:url(:/StarLightStation/icon/ruler-no.png)}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CParonamaSourceImageDisplayerWidget</class>
   <extends>QGraphicsView</extends>
   <header>paronamasourceimagedisplayerwidget.h</header>
  </customwidget>
  <customwidget>
   <class>CParonamaCPRImageDisplayerWidget</class>
   <extends>QGraphicsView</extends>
   <header>paronamacprimagedisplayerwidget.h</header>
  </customwidget>
  <customwidget>
   <class>CParonamaSagittalImageDisplayerWidget</class>
   <extends>QGraphicsView</extends>
   <header>paronamasagittalimagedisplayerwidget.h</header>
  </customwidget>
  <customwidget>
   <class>CParonamaVRImageDisplayerWidget</class>
   <extends>QGraphicsView</extends>
   <header>paronamavrimagedisplayerwidget.h</header>
  </customwidget>
  <customwidget>
   <class>CIconButton</class>
   <extends>QPushButton</extends>
   <header location="global">iconbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="StarLightStation.qrc"/>
 </resources>
 <connections/>
</ui>
